pkg/monitoring/alloy/testdata/alloy_config_multitenants.river

```diff
@@ -19,13 +23,36 @@ prometheus.operator.servicemonitors "default" {
   }
 }
 
-prometheus.operator.podmonitors "default" {
-  forward_to = [prometheus.remote_write.default.receiver]
+prometheus.operator.podmonitors "giantswarm_legacy" {
+  forward_to = [prometheus.remote_write.giantswarm.receiver]
```

Comment:
This receiver is non-existent in this file. I get that it "should" be created because the giantswarm tenant should exists. But this has a high potential of breaking the alloy config.

I would prefer when we move this behavior and logic into the operator, ideally configurable via a flag, rather than making an assumption here in the config.

Reply:
Something like:
- Add a boolean variable to the template to create legacy service and pod monitor for the given tenant
- Set the variable to true from operator for the giantswarm tenant

Reply:
Then the tenant list would move from a list of strings to a list of struct. How about we move them to the loop and only rendre them if the tenant is giantswarm?

Reply:
@TheoBrigitte what do you think now?